spec/concepts/todo/create_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 RSpec.describe Todo::Create do
   it "can be spec'd" do
-    raise "Implement me"
+    todo = Todo::Create.(todo: {title: "Sample title"}).model
+    expect(todo.title).to eq("Sample title")
   end
```

Comment:
Hi Chanel, can we describe what Todo::Create does using this spec file?

e.g.
- what happens when a name is specified?
- what happens when it's not?